scripts/generate_tables.py

```diff
@@ -0,0 +1,104 @@
+import os, re
+import numpy as np
+import pandas as pd
+
+
+CONTRASTS = {
+    "t1map": ["T1map"],
+    "mp2rage": ["inv-1_part-mag_MP2RAGE", "inv-2_part-mag_MP2RAGE"],
+    "t1w": ["T1w", "space-other_T1w", "acq-lowresSag_T1w"],
+    "t2w": ["T2w", "space-other_T2w", "acq-lowresSag_T2w", "acq-highresSag_T2w"],
+    "t2star": ["T2star", "space-other_T2star"],
+    "dwi": ["rec-average_dwi", "acq-dwiMean_dwi"],
+    "mt-on": ["flip-1_mt-on_space-other_MTS", "acq-MTon_MTR"],
+    "mt-off": ["flip-2_mt-off_space-other_MTS"],
+    "unit1": ["UNIT1"],
+    "psir": ["PSIR"],
+    "stir": ["STIR"]
+}
```

Comment:
Very minor comment.
This dict looks similar/identical to: https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/f9e58332434a11ebb798bcdc4d0e99dfa1663933/monai/main.py#L29-L41
For future scripts and PRs, you could maybe define such common dicts in a utils script and import them.